### PR TITLE
Updated MindsDB

### PR DESCRIPTION
**Pull request by chandrevdw31**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

---
### Modify your CI run:
**NOTE:** If your merge the PR with modified CI you **MUST KNOW** what you are doing
**NOTE:** Checked options will be applied if set before CI RunConfig/PrepareRunConfig step

#### Include tests (required builds will be added automatically):
- [ ]  Fast test
- [ ]  Integration Tests
- [ ]  Stateless tests
- [ ]  Stateful tests
- [ ]  Unit tests
- [ ]  Performance tests
- [ ]  All with ASAN
- [ ]  All with TSAN
- [ ]  All with Analyzer
- [ ]  Add your option here

#### Exclude tests:
- [ ]  Fast test
- [ ]  Integration Tests
- [ ]  Stateless tests
- [ ]  Stateful tests
- [ ]  Performance tests
- [ ]  All with ASAN
- [ ]  All with TSAN
- [ ]  All with MSAN
- [ ]  All with UBSAN
- [ ]  All with Coverage
- [ ]  All with Aarch64
- [ ]  Add your option here

#### Extra options:
- [ ]  do not test (only style check)
- [ ]  disable merge-commit (no merge from master before tests)
- [ ]  disable CI cache (job reuse)

#### Only specified batches in multi-batch jobs:
- [ ]  1
- [ ]  2
- [ ]  3
- [ ]  4

### CODE REVIEW

**Comment by robot-ch-test-poll4** [comment]
*This is an automated comment for commit e1fc3e6b6c193996751c0f2189271d4444320c42 with description of existing statuses. It's updated for the latest CI running*

[❌ Click here](https://s3.amazonaws.com/clickhouse-test-reports/62910/e1fc3e6b6c193996751c0f2189271d4444320c42/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td><a href="https://github.com/ClickHouse/ClickHouse/blob/master/.github/PULL_REQUEST_TEMPLATE.md?plain=1">❌ failure</a></td></tr>
<tbody>
</table>

File: docs/en/interfaces/third-party/integrations.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ ClickHouse, Inc. does **not** maintain the tools and libraries listed below and
         - [clickhouse-maxmind-geoip](https://github.com/AlexeyKupershtokh/clickhouse-maxmind-geoip)
 - AutoML
     - [MindsDB](https://mindsdb.com/)
-        - [MindsDB](https://github.com/mindsdb/mindsdb) - Predictive AI layer for ClickHouse database.
+        - [MindsDB](https://github.com/mindsdb/mindsdb) - The platform for customizing AI from enterprise data.
```
Comment (alexey-milovidov) [inline docs/en/interfaces/third-party/integrations.md:79]: How is it related to ClickHouse?

**Comment by alexey-milovidov** [comment]
Either make it clear that this is a ClickHouse integration or feel free to remove it entirely.

File: docs/en/interfaces/third-party/integrations.md (hunk 1 above)
Comment (chandrevdw31) [reply docs/en/interfaces/third-party/integrations.md]: Thank you @alexey-milovidov, I have updated it. Can you kindly have a look?

**Review by alexey-milovidov — APPROVED** [review]

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**